entrypoint.sh

```diff
@@ -1238,17 +1238,48 @@ function prepare_gcc() {
 
     ALT_GCC_PRIO=200
     proc_version=$(cat /proc/version)
```

Comment:
Could you add in a comment examples of the file content for different OSes/versions?

Reply:
Added example outputs